docs/CONTRIBUTING.md

```diff
@@ -35,24 +35,34 @@ kubectl-gadget-linux-amd64` or `make kubectl-gadget-darwin-amd64`.
 
 ### Building the gadget container image
 
+Inspektor Gadget provides two different container images:
+- gadget-bcc: Contains CO-RE and BCC gadgets
+- gadget-core: Containes only CO-RE gadgets that are integrated with the gadget tracer manager
```

Comment:
Given that `gadget-bcc` contains both bcc-based gadgets and core-based gadgets, it is a bit confused to call it bcc only. What about calling the two files:
1. `gadget-default`
2. `gadget-coreonly`

(and renaming the Dockerfiles accordingly)

Reply:
Renamed it to be `gadget-default` and `gadget-core`.